### PR TITLE
DDF-2385 add banner warning to admin console when user session expires

### PR DESCRIPTION
**Pull request by ryeats**
#### What does this PR do?
Adds a warning banner if the user session expires
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@tbatie @adimka @djblue 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@shaundmorris
@stustison
#### How should this be tested?
log in to the admin console with two tabs logout with one of them and then try to do some action on the other and verify the banner changes to a warning about being logged out.

#### Screenshots (if appropriate)
screen shot 2016-08-11 at 8 44 02 am
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by tbatie** [comment]
:+1:

File: platform/admin/ui/src/main/webapp/js/util/SessionRefresherUtil.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, document, setInterval*/
+define(['jquery'
+], function ($) {
+    "use strict";
+
+    var Util = {
```
Comment (djblue) [inline platform/admin/ui/src/main/webapp/js/util/SessionRefresherUtil.js]: Maybe we can return the object here.

**Comment by shaundmorris** [comment]
@lessarderic this is the PR on giving a warning for a session timeout in the Admin UI

**Comment by shaundmorris** [comment]
Tested manually, looks good. Is it possible to make the "show details" part not show up? It just says "forbidden". If not a big deal. 👍

**Comment by ryeats** [comment]
changed the description from forbidden to 
screen shot 2016-08-18 at 3 45 42 pm

**Comment by shaundmorris** [comment]
👍

**Comment by shaundmorris** [comment]
Is there a plan to give the administrator the option to extend the session before it times out? Like a countdown or a popup that ask if they want to extend. Just thinking of other applications I have before the session times out.

**Comment by stustison** [comment]
@shaundmorris the reason why we can't do any sort of popup is because that would tie our UI pretty closely to how it is being secured. The advantage of the website you're thinking of (probably bank websites) is that they don't care and they can couple the UI to the way it is secured as much as they like. Our situation is a little different.

**Comment by stustison** [comment]
👍

**Comment by tbatie** [comment]
Manually tested with new changes :+1:

**Comment by shaundmorris** [comment]
Did we plan on back porting this one?

**Comment by ryeats** [comment]
Can't it uses the new taxonomy

**Pull request merged**